src/test/java/io/dockstore/provision/S3CmdPluginTest.java

```diff
@@ -0,0 +1,87 @@
+package io.dockstore.provision;
+
+import java.io.File;
+import java.lang.reflect.Method;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author gluu
+ * @since 07/03/17
+ */
+public class S3CmdPluginTest {
+    S3CmdPlugin.S3CmdProvision icgcGetProvision;
+    final String resourcesDirectory = new File("src/test/resources").getAbsolutePath();
+    @Before
+    public void before() throws Exception {
+        icgcGetProvision = new S3CmdPlugin.S3CmdProvision();
+        setConfiguration();
+    }
+
+    private void setConfiguration() throws Exception {
+        HashMap hm = new HashMap();
+        hm.put("client", "/home/travis/.local/bin/s3cmd");
+//        hm.put("client", "/usr/local/bin/s3cmd");
```

Comment:
Does this have to be manually uncommented to run locally? 
If yes, just use https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#exists(java.nio.file.Path,%20java.nio.file.LinkOption...)
If no (i.e. this is superfluous, just delete it)

Reply:
It depends on where s3cmd is installed.  I happen to have mine installed in `/usr/local/bin/s3cmd`.  Travis by default installs it in `/home/travis/.local/bin/s3cmd`  instead.  If the local user doesn't even have s3cmd installed, the tests would fail also.

Reply:
Yeah, I would just check the two locations using Files.exists and fail otherwise indicating just that